.circleci/config.yml

```diff
@@ -0,0 +1,497 @@
+            # or
+            # - DOCKER_TAG: 1.1.2 (Git tag v1.1.2)
+            echo "DOCKER_TAG: ${DOCKER_TAG} (Git ${RELEASE_TYPE}${CIRCLE_TAG})"
+            docker push fundocker/mork:${DOCKER_TAG}
+            if [[ -n "$CIRCLE_TAG" ]] && [[ "$CIRCLE_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              docker push fundocker/mork:latest
+            fi
+
+  # Make a new github release
+  release:
+    docker:
+      - image: cimg/base:current
+        auth:
+          username: $DOCKER_HUB_USER
+          password: $DOCKER_HUB_PASSWORD
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - *docker-login
+      - attach_workspace:
+          at: ~/fun
+      - run:
+          name: Install gh CLI
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | \
+              sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | \
+              sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            sudo apt update
+            sudo apt install gh
+      - run:
+          name: Define release reference
+          command: |
+            echo 'RELEASE=$(echo "${CIRCLE_TAG}")' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Get release changes
+          command: |
+            tag="${RELEASE/v/}" ;
+            sed -E -n "/^## \[${tag}\]/,/^##\ /{/^## \[${tag}\]/d ;/^##\ /d; p}" CHANGELOG.md > release.md
+      - run:
+          name: Create release on GitHub
+          command: |
+            gh config set prompt disabled
+            if ! gh release list | grep "${RELEASE}"; then
+              gh release create -F release.md -t "${RELEASE/v/}" "${RELEASE}";
+            fi
+workflows:
+  version: 2
+
+  mork:
+    jobs:
+      # Git jobs
+      #
+      # Check validity of git history
+      - lint-git:
+          filters:
+            tags:
+              only: /.*/
+      # Check CHANGELOG update
+      - check-changelog:
+          filters:
+            branches:
+              ignore: main
+            tags:
+              only: /(?!^v).*/
+      - lint-changelog:
+          filters:
+            branches:
+              ignore: main
+            tags:
+              only: /.*/
+      # Check Renovate
+      - check-renovate-configuration:
+          filters:
+            tags:
+              only: /.*/
+
+      # Docker jobs
+      #
+      # Build images
+      - build-docker:
+          filters:
+            tags:
+              only: /.*/
+
+      # Backend jobs
+      #
+      # Build, lint and test production and development Docker images
+      # (debian-based)
+      - build:
+          matrix:
+            parameters:
+              python-image: [python:3.8, python:3.9, python:3.10, python:3.11, python:3.12]
```

Comment:
Do we really want to test against all of this version ? IMO it is useless, the production image will be build with python 3.12, we can have all the benefit and feature available with python 3.12